docker/tomcat-members-debug.yml

```diff
@@ -0,0 +1,28 @@
+version: ''
+
+name: tomcat
+
+services:
+  ff-test:
+    volumes:
+      - "../test/src/test/testtool:/opt/frank/testtool-ext"
+    image: private.docker.nexus.frankframework.org/ff-test:${VERSION:-latest}-tomcat
+    build: Tomcat/test
+    ports:
+      - "8080:8080"
+      - "5700-5709:5700-5709"
+    environment:
+      larva.adapter.active: false
+      management.gateway.inbound.class: org.frankframework.management.gateway.HazelcastInboundGateway
+      management.gateway.outbound.class: org.frankframework.management.gateway.HazelcastOutboundGateway
+  ff-test2:
+    volumes:
+      - "../test/src/test/testtool:/opt/frank/testtool-ext"
+    image: private.docker.nexus.frankframework.org/ff-test:${VERSION:-latest}-tomcat
+    build: Tomcat/test
+    ports:
+      - "8081:8080"
+      - "5710-5720:5700-5710"
```

Comment:
Waarom is de source port range hier 5700-5710 terwijl die in het andere image 5700-5709 is? Zouden beide images niet port 5710 kunnen gebruiken intern en dus moeten mappen naar een andere port voor buitenaf?

Reply:
Docker kan er niet goed tegen als je de zelfde poorten gebruikt, op zich is 1 poort forwarden genoeg, we of-setten alles met 10 om het op de docker host goed te krijgen.

Reply:
Ik snap dat je alles met 10 wil offsetten maar blijf het raar vinden dat de source ranges niet hetzelfde zijn. 😃

Reply:
De poorten op de containers zijn het zelfde..

Reply:
#De waarde rechts van `:`

Reply:
> Ik twijfel ook een beetje of het wel werkt. De Docker container kan zomaar een poort openen, ook als die niet exposed wordt. Dus misschien kan het fout gaan als hazelcast een poort kiest die niet exposed is.

Ja maar hij begint onderaan, en het lijkt mij sterk dat je in een test omgeving meer dan 10 dingen hebt draaien toch?

Reply:
> De waarde rechts van `:`

Dan zijn de sources ranges dus wel het zelfde ;)